Kinvey.Core/Push/AbstractPush.cs

```diff
@@ -84,20 +84,23 @@ public class RemovePush : AbstractKinveyClientRequest<PushPayload> {
 			}
 		}
 
-
 		[JsonObject(MemberSerialization.OptIn)]
 		public class PushPayload : JObject{
 
 			[JsonProperty]
-			private String platform { get; set;} 
+			private string platform { get; set;} 
 
 			[JsonProperty]
-			private String deviceId {get; set;}
+			private string token {get; set;}
+
+            [JsonProperty]
+            private string service { get; set; }
 
-			public PushPayload(string platform, string deviceId) {
+            public PushPayload(string platform, string token) {
 				this.platform = platform;
-				this.deviceId = deviceId;
-			}
+				this.token = token;
+                this.service = "firebase";
```

Comment:
this should be only for `Android`. Have you test this against `iOS`?

Reply:
I will correct the code according to your request and check how it works for iOS.